### PR TITLE
Fix daylight shift test

### PR DESCRIPTION
**Pull request by saifalharthi**
This PR contains a simple fix for the failing test in this issue #141

### CODE REVIEW

**Comment by randallwhitman** [comment]
Would it be simpler to choose a time near noon instead of a time near midnight?
Versus, would choosing a different rather-random epoch number, merely change which timezone suffers the test failure?
@saifalharthi

**Comment by saifalharthi** [comment]
The problem was daily light shifting. It would've been easier to select another random epoch number but we'll still have some problems with respect to other timezones. So, I forced the return date to be converted and formatted to a timezone that has daylight savings and that should work with any epoch number given.
@randallwhitman

**Comment by randallwhitman** [comment]
So in other words, no what epoch-number we choose, it will be too close to midnight in some time zone.
Thanks Saif.

File: hive/src/test/java/com/esri/hadoop/hive/serde/TestEsriJsonSerDe.java [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
 
-import com.esri.core.geometry.Geometry;
-import com.esri.core.geometry.Point;
-import com.esri.core.geometry.SpatialReference;
-import com.esri.core.geometry.ogc.OGCGeometry;
-import com.esri.hadoop.shims.HiveShims;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.TimeZone;
```
Comment (randallwhitman) [inline hive/src/test/java/com/esri/hadoop/hive/serde/TestEsriJsonSerDe.java]: Any chance of undoing the spurious re-ordering?

**Comment by randallwhitman** [comment]
Saif - I reviewed TestGeoJsonSerDe looking if it would likely need the same change.  It looks like the timezone was addressed in a different manner in that one.  Do you think either way is any better or worse than the other?

**Comment by saifalharthi** [comment]
I will take a look at it

**Comment by saifalharthi** [comment]
I think the approach in TestGeoJsonSerDe should work out fine. I think since it deals with a default timezone offset it will guarantee a more stable time conversion.

**Comment by randallwhitman** [comment]
OK, sounds good.

**Comment by randallwhitman** [comment]
Well, if it works, we can merge now and later think about consistency of timezone handling in that test for EsriJson & GeoJson.

**Pull request merged**